src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Clients/ActorClient.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Net.Http;
+using Altinn.Platform.Authorization.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Authorization.Clients
+{
+    public class ActorClient
+    {
+        private readonly GeneralSettings _generalSettings;
+
+        /// <summary>
+        /// Gets an instance of httpclient from httpclientfactory
+        /// </summary>
+        public HttpClient Client { get; }
+
+        public ActorClient(HttpClient client, IOptions<GeneralSettings> generalSettings)
+        {
+            _generalSettings = generalSettings.Value;
+            Client = client;
+            Client.BaseAddress = new Uri(_generalSettings.GetBridgeApiEndpoint);
+            Client.Timeout = new TimeSpan(0, 0, 30);
+            Client.DefaultRequestHeaders.Clear();
```

Comment:
The client is only being used for a single thing at this point. Why not set the application/json at this point rather than in the wrapper?

Reply:
fixd